gobblin-runtime/src/main/java/org/apache/gobblin/runtime/api/MysqlMultiActiveLeaseArbiter.java

```diff
@@ -218,52 +213,28 @@ public MysqlMultiActiveLeaseArbiter(Config config) throws IOException {
       throw new IOException("Table creation failure for " + leaseArbiterTableName, e);
     }
     initializeConstantsTable();
-    runRetentionOnArbitrationTable();
+
+    // Periodically deletes all rows in the table with event_timestamp older than the retention period defined by config.
+    mySQLStoreUtils.runSqlCommandWithInterval(thisTableRetentionStatement, 4, TimeUnit.HOURS);
+
     log.info("MysqlMultiActiveLeaseArbiter initialized");
   }
 
   // Initialize Constants table if needed and insert row into it if one does not exist
   private void initializeConstantsTable() throws IOException {
     String createConstantsStatement = String.format(CREATE_CONSTANTS_TABLE_STATEMENT, this.constantsTableName);
-    withPreparedStatement(createConstantsStatement, createStatement -> createStatement.executeUpdate(), true);
+    mySQLStoreUtils.withPreparedStatement(createConstantsStatement, createStatement -> createStatement.executeUpdate(),
```

Comment:
seeing this use--which is quite reasonable IMO--makes me wonder whether this ought to be a common base class.  what are the args for vs. against that approach?  are you concerned about multiple inheritance, since some mysql store classes need another different base class?